crates/bevy_ecs/src/system/commands/mod.rs

```diff
@@ -951,7 +951,7 @@ impl EntityCommands<'_> {
 
     /// Adds a [`Bundle`] of components to the entity without overwriting.
     ///
-    /// This is the same as [`Commands::insert`], but in case of duplicate components
+    /// This is the same as [`insert`], but in case of duplicate components
```

Comment:
This was changed back after being fixed in fc90351002bd0673ed7045f7c4e313290e37f0eb and is the reason CI fails